Documentation/git-rebase.txt

```diff
@@ -378,8 +378,12 @@ default is `--no-fork-point`, otherwise the default is `--fork-point`.
 See also INCOMPATIBLE OPTIONS below.
 
 --committer-date-is-author-date::
+	Committer date is the date on which the commit was created or
+	modified. Override this by using the same value as the author
+	date. This implies --force-rebase for interactive / merge backend.
```

Comment:
The first sentence appears to be a description of what committer date normally means, but is easy to read as the intent of the new flag, making it confusing.  Also, not sure why we need to describe what committer date normally means here.  And why is --force-rebase only implied for the interactive backend and not the 'am' one?

My crack at the description would be something like:
"Instead of recording the time the rebased commits are created as the committer date, reuse the author date as the committer date."

...and maybe also add comments about --force-rebase, but make the two backends the same first.  We've put a lot of work into making the backends more consistent and we don't want to go backwards on that.

Reply:
Just realized, it implies `-f` for both rebases :sweat_smile:.